src/turing.jl

```diff
@@ -0,0 +1,83 @@
+function from_turing(
+    chns=nothing;
+    model=nothing,
+    rng=Random.default_rng(),
+    nchains=ndraws = chns isa Turing.MCMCChains.Chains ? last(size(chns)) : 1,
+    ndraws=chns isa Turing.MCMCChains.Chains ? first(size(chns)) : 1_000,
+    library=Turing,
+    observed_data=nothing,
+    constant_data=nothing,
+    posterior_predictive=nothing,
+    prior=nothing,
+    prior_predictive=nothing,
+    log_likelihood=nothing,
+    kwargs...,
+)
+    groups = Dict{Symbol,Any}(
+        :observed_data => observed_data,
+        :constant_data => constant_data,
+        :posterior_predictive => posterior_predictive,
+        :prior => prior,
+        :prior_predictive => prior_predictive,
+        :log_likelihood => log_likelihood,
+    )
+    model === nothing && return from_mcmcchains(chns; library=library, groups..., kwargs...)
+    if groups[:prior] === nothing
```

Comment:
I'm not certain I would make sampling from the prior set by default, as it can be costly :confused:
At least personally, when I call a "constructor" like this, I don't expect it to do more computation unless I explicitly tell it to. 

So IMO at least there should be a way of saying "please also compute prior" or "please don't", yah know?